.gitignore

```diff
@@ -8,3 +8,5 @@ setup.cfg
 /dist/
 /venv/
 *.swp
+/apluslms_roman/*.yml
```

Comment:
This line shouldn't be here. There is no need to ignore yaml files in that path.

Reply:
I was thinking to ignore roman_config.yml

Reply:
It doesn't belong in that folder in the first place :)